packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -136,15 +136,27 @@ module.exports = {
        * typically include will not be specified that way
        *
        * @type {{
-          extends: string[];
+          extends: string;
           include: string[];
-          excludes: string[];
+          exclude: string[];
           compilerOptions: Record<string,unknown>;
           references?: Array<{path:string}>
           }}
        */
     const tsconfig = jju.parse(fs.readFileSync(tsconfigPath).toString());
 
+    /**
+     * Handle any necessary extends merging here, make sure to treat just like native tsconfigs would
+     */
+    if (tsconfig.extends) {
+      const parentTsConfigPath = path.join(path.dirname(tsconfigPath), tsconfig.extends);
```

Comment:
Let's add a comment about why in the end we didn't end up using the TS API to do this